README.md

```diff
@@ -462,8 +462,8 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
 Many people all over the world have helped make this project better.  You'll want to check out:
 
 * [What are some good first issues for new contributors to the repo?](https://github.com/azure/azure-sdk-for-c/issues?q=is%3Aopen+is%3Aissue+label%3A%22up+for+grabs%22)
-* [How to build and test your change](https://github.com/Azure/azure-sdk-for-c/blob/master/CONTRIBUTING.md#developer-guide)
-* [How you can make a change happen!](https://github.com/Azure/azure-sdk-for-c/blob/master/CONTRIBUTING.md#pull-requests)
+* [How to build and test your change](./CONTRIBUTING.md#developer-guide)

```

Comment:
The unfortunate side effect of such a change is that more links behave unexpectedly in doxygen.

See this issue which might suggest we go in the opposite direction and make all links absolute:
https://github.com/Azure/azure-sdk-for-c/issues/746

I understand that not having links work in a particular branch is a contributor pain point, but broken links hurts the customer experience. If we can resolve both issues, great, but if not, I would focus on improving user experience and readability of docs.

Reply:
Replied in the GH issue